packages/core/lib/commands/run/run.js

```diff
@@ -1,61 +1,33 @@
-const path = require("path");
-const originalRequire = require("original-require");
 const TruffleError = require("@truffle/error");
```

Comment:
Would this file be better renamed "execute" or something? A small thing but `run/run` feels a bit a bit bad to me. What do you think?